### ISSUE TITLE
Infinite loading when opening any apk file

### ISSUE THREAD

**Issue by Younes-L** [MuntashirAkon/AppManager#291]
**Describe the bug**
trying to open "App Info" or "installer" for an .apk file results in infinite loading.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any app that can browse browse files, example : system _Files_ app
2. Find an .apk file and click on it
3. When prompted with the "open with" menu, choose AM's `App Info` or `Install`

**Expected behavior**
After a fraction of a second, AM displays info of the apk

**Screenshots**
![Screenshot_20210221-092636_cropped](https://user-images.githubusercontent.com/23421600/108621062-b347a000-7430-11eb-8336-0ccec14f6e72.png)

**Crash logs**
The following entries can be found in `am.log` when trying to open a vlc.apk for example :
```
31:22.860 D/ADVM: On Clear called for io.github.muntashirakon.AppManager.debug 
31:33.799 D/ADVM: New constructor called. 
31:33.906 D/ADVM: Package Uri is being set 
31:33.969 D/ADVM: Package name is being set for org.videolan.vlc
```

**Device info**
 - Device: Google Pixel 2
 - OS Version: Android 9
 - App Manager Version: 2.5.24-DEBUG `#905` and `#917` (latest)
 - Mode: no-root

**Additional context**
in my case the newest working build is `#878`.
However starting build `#881` I'm facing another problem to be reported in separate issue if needed

Found related issue : #223

**Comment by MuntashirAkon** [comment]
#223 was about loading the main page. Take logs using logcat. Due to various complications, `am.log` has become obsolete and will be removed in future. Remember to restart your phone before taking the logs.

**Comment by Younes-L** [comment]
Thanks for clarification, I will make sure to provide logcat when pc is working again 👍 
Meanwhile AM `881` gives more useful info in am.log for doing the same steps (with error : `could not fetch package info` )  , here it is :
```
0:12:03.239 D/ADVM: New constructor called. 
0:12:03.354 D/ADVM: Package Uri is being set 
0:12:03.407 D/ADVM: Package name is being set for org.videolan.vlc 
0:12:03.415 E/ADA: Could not fetch package info. android.os.RemoteException: AMService not running.
  at io.github.muntashirakon.AppManager.ipc.IPCUtils$AMServiceConnectionWrapper.getServiceSafe(IPCUtils.java:156)
  at io.github.muntashirakon.AppManager.ipc.IPCUtils$AMServiceConnectionWrapper.getAmService(IPCUtils.java:143)
  at io.github.muntashirakon.io.ProxyOutputStream.<init>(ProxyOutputStream.java:41)
  at io.github.muntashirakon.AppManager.utils.IOUtils.saveZipFile(IOUtils.java:162)
  at io.github.muntashirakon.AppManager.utils.IOUtils.saveZipFile(IOUtils.java:153)
  at io.github.muntashirakon.AppManager.apk.ApkFile$Entry.getRealCachedFile(ApkFile.java:692)
  at io.github.muntashirakon.AppManager.details.AppDetailsViewModel.setPackage(AppDetailsViewModel.java:131)
  at io.github.muntashirakon.AppManager.details.AppDetailsActivity.lambda$onAuthenticated$5$AppDetailsActivity(AppDetailsActivity.java:102)
  at io.github.muntashirakon.AppManager.details.-$$Lambda$AppDetailsActivity$N44Vyz4IkC88KWOwbq04gs3biVs.run(Unknown Source:14)
  at java.lang.Thread.run(Thread.java:764) 
```

**Comment by MuntashirAkon** [comment]
Thanks. You won't need to send me any logs.

**Comment by MuntashirAkon** [comment]
Fixed in 87b3a47e
